source/Octopus.Client/Model/TenantResource.cs

```diff
@@ -64,5 +64,9 @@ public TenantResource ClearProjects()
         }
 
         public string SpaceId { get; set; }
+        
+        public string Description { get; set; }
+        
+        public string ClonedFromTenantId { get; set; }
```

Comment:
Should `CloneFromTenantId` be read-only?

Reply:
How does one do that?

Reply:
I tried changing it to:
```diff
-        public string ClonedFromTenantId { get; set; }
+        public string ClonedFromTenantId { get; }
```

and it didn't work... came through as null.

Reply:
I think if you add the `WriteableOnCreate` attribute it'll make it readonly once created

Reply:
although, the `ProjectResource` also has a `ClonedFrom` property which is **not** readonly... https://github.com/OctopusDeploy/OctopusClients/blob/master/source/Octopus.Client/Model/ProjectResource.cs#L56

Reply:
> I think if you add the `WriteableOnCreate` attribute it'll make it readonly once created

This is done on the server side PR - https://github.com/OctopusDeploy/OctopusDeploy/pull/4285/files#diff-51caff6d0c4dcb6968bdcf3f0ea615ddR70. Not sure it should be done here though?
🤔
Actually, looking through the code - lots of other things have `[WriteableOnCreate]` here... I'm going to stick it on.